### ISSUE TITLE
Kubernetes' `VolumeGroupSnapshot` API support

### ISSUE THREAD

**Issue by gbartolini** [cloudnative-pg/cloudnative-pg#4680]

**Comment by gbartolini** [comment]
Moving this to 1.26.0 as the feature is still alpha in Kubernetes and Leonardo is working on it.

**Comment by gbartolini** [comment]
Leonardo has updated the patch and made it good enough for demonstration purposes. As we approach 1.26 and wait for further implementations of CSI volume group snapshots in the Kubernetes storage area, I think we should postpone this feature to version 1.27.0 for now. What do the other maintainers think?

**Comment by sxd** [comment]
Yeah, for sure, even now we have some delay on the versions CSI packages and images so is better to wait until we can test everything as much as we can, 1.27.0 looks ok to me

**Comment by mnencia** [comment]
I agree with @gabrielebartolini's assessment. We will continue monitoring the maturity and adoption of the VolumeGroupSnapshot API and revisit integration for 1.27.0.

**Comment by leonardoce** [comment]
Yep, let's move it to 1.27.0.

**Comment by fcanovai** [comment]
+1 on postponing

**Comment by gbartolini** [comment]
Blocked by #7341 - moving this to later.

**Comment by gbartolini** [comment]
@leonardoce what do you think we should do with this issue? Shall we postpone this to 1.29 and after we have made a decision on #7341?

**Comment by gbartolini** [comment]
I am moving this to 1.29 for now, given that we are approaching the 1.28 release.
